### ISSUE TITLE
Issue installing chrome stable on Ubuntu 14.04

### ISSUE THREAD

**Issue by mhemesath** [dhoer/chef-chrome#2]
When running the default recipe on Ubuntu 14.04, I'm getting this error:

```sh
 STDERR: E: There are problems and -y was used without --force-yes
       ---- End output of apt-get -q -y install google-chrome-stable=41.0.2272.101-1 ----
       Ran apt-get -q -y install google-chrome-stable=41.0.2272.101-1 returned 100
```

**Comment by dhoer** [comment]
Open a terminal and enter the following:
```
wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | sudo apt-key add -
```
That will import Google's updated GPG Key.

This solution came from: http://www.google.com/linuxrepositories/

**Comment by mhemesath** [comment]
Shouldn't this be part of the recipe if its required?

**Comment by dhoer** [comment]
You bet.  The above is the work around until I get this added.  Google just updated their key last week causing this issue on Ubuntu platforms: https://www.google.com/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=ubuntu+google%27s+updated+gpg+key

**Comment by dhoer** [comment]
It should now be patched.

**Comment by mhemesath** [comment]
That worked, thanks!

**Comment by ramonskie** [comment]
the patch dit not work for me
i still need to manually do 
```
wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | sudo apt-key add -
```

**Comment by dhoer** [comment]
Is it randomly working?  Because I see people complaining about that as well.

**Comment by ramonskie** [comment]
run it a few times
it seems that the execute doesn't even run

**Comment by dhoer** [comment]
Thanks, It is because execute needs to run at compile time.  Which it isn't.  I wonder how test-kitchen ran Ubuntu successfully?  Let me fix that.

**Comment by ramonskie** [comment]
if i didn't make any typos this should run at compile time
```
execute 'update_sing_key' do
  command 'wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | sudo apt-key add -'
 action :nothing
end.run_action(:run)
```

**Comment by dhoer** [comment]
After spending some time on this.  I think there is an issue with google-chrome-stable=41.0.2272.101-1.  The fact that the patch worked was random chance.  I ran Ubuntu 12 and 14 multiple times, and succeeds or fails randomly with or without the patch.   If I had to guess, one of Google's servers has a corrupt 41.0.2272.101-1 package.  There is not much I can do on my end.  I'm not even sure how to report this issue to Google.
